bindings/node.js/lib/kzg.d.ts

```diff
@@ -6,7 +6,6 @@ export type Bytes32 = Uint8Array; // 32 bytes
 export type Bytes48 = Uint8Array; // 48 bytes
 export type KZGProof = Uint8Array; // 48 bytes
 export type KZGCommitment = Uint8Array; // 48 bytes
-export type Blob = Uint8Array; // 4096 * 32 bytes
```

Comment:
Is the idea that `blob` params everywhere will be untrusted length so reverting to the base `Uint8Array` for param type?

Reply:
Hmm thanks for pointing this out. I think I want to keep this type now.

Reply:
Yeah, added this back. This is now the same as the other bindings.